secp256k1jni/src/main/java/org/bitcoin/NativeSecp256k1.java

```diff
@@ -52,7 +52,7 @@ public class NativeSecp256k1 {
      * @param pub The public key which did the signing
      */
     public static boolean verify(byte[] data, byte[] signature, byte[] pub) {
-        checkArgument(data.length == 32 && signature.length <= 520 && pub.length <= 520);
+        checkArgument(signature.length <= 520 && pub.length <= 520);
```

Comment:
I believe the ECDSA expects 32 byte message. It's only the Schnorr signing/verify doesn't restrict the message length: https://github.com/bitcoin-core/secp256k1/blob/70f149b9a1bf4ed3266f97774d0ae9577534bf40/src/secp256k1.c#L566